antsibull/cli/doc_commands/sphinx_init.py

```diff
@@ -0,0 +1,96 @@
+# Author: Felix Fontein <felix@fontein.de>
+# License: GPLv3+
+# Copyright: Ansible Project, 2021
+"""Entrypoint to the antsibull-docs script."""
+
+import os
+import os.path
```

Comment:
Having this project py3-only, I recommend using pathlib internally.

Reply:
Since we're using `os.path` everywhere and `pathlib` nowhere, I tend to stick to `os.path` for this PR.

Reply:
We'll probably rewrite the whole directory/file handling code to involve file descriptors to avoid some security issues (see yesterday's discussion in #ansible-docs).